.travis.yml

```diff
@@ -12,11 +12,11 @@ env:
     - secure: "UVIbeoJpx9n0KSWHbpdWZuR4zeZfXw3JQBq2Zb1h0/j4uREfXWiGFuNVQBPjsdtYi2+LkecjJ/MgKWo+SS0SjsvTa/ZPkOXt4408H1qYgwVnvWoeL/g1QQ6T8ADNysl/4ZtJLnSp0WXYEBkfHYZJD++vgDwkpVhnCLVzSFVAHls="
 
 before_install:
-  - export JDK="openjdk11"
+  - export JDK="openjdk14"
```

Comment:
How is changing to java 14 related to this.

Reply:
Because I had various issues and tried if JavaFX 14 solves those. They didn't, but it seemed pointless to me to go back to 11 just to go forward to 14 later.

Plus, they fixed [Win: UNDECORATED windows are not minimized with the taskbar button](https://bugs.openjdk.java.net/browse/JDK-8088717) in 13 so I was able to remove our workaround (See #31).

Reply:
Actually I could not find any issue so I think it is fine.